docker-compose.yml

```diff
@@ -16,24 +16,65 @@ services:
     image: postgis/postgis:13-3.2-alpine
     volumes:
       - pg_data:/var/lib/postgresql/data
+      # copy the sql script to create db/tables
+      - ./sql/init_db.sql:/docker-entrypoint-initdb.d/init_db.sql
     environment:
-      - POSTGRES_PASSWORD=postgres
-      - POSTGRES_USER=postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_USER: postgres
+    restart: always
     ports:
       - 5432:5432
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U postgres"]
+      start_period: 60s
+      interval: 5s
+      timeout: 3s
+      retries: 10
 
   api_service:
     container_name: api_service
     build:
         context: .
-        dockerfile: Dockerfile
+        dockerfile: Dockerfile.ecs
     image: wcs/mermaid-api:latest
-    volumes:
-      - $PWD/src:/var/projects/webapp
     ports:
-      - 8080:8080
+      - 8081:8081
+    environment:
+      # db
+      DB_NAME: mermaid
+      DB_HOST: api_db
+      DB_PORT: 5432
+      DB_USER: postgres
+      DB_PASSWORD: postgres
+
+      # Auth0
+      MERMAID_DOMAIN: datamermaid.auth0.com
+      MERMAID_API_AUDIENCE: https://dev-api.datamermaid.org
+      AUTH0_MANAGEMENT_API_AUDIENCE: https://datamermaid.auth0.com/api/v2/
+
+      # django
+      ENV: local
+      RESTORE: local
+      BACKUP: local
+      DEBUG: "True"
+
+      DEFAULT_DOMAIN_API: api_service
```

Comment:
Should this be `api_runserver`?

Reply:
In the context of local development, I don't think it matters... In fact, it should probably be `localhost` since it's used when sending an admin URL via email... so `api_runserver` DNS is not even reachable outside of the docker network.

Reply:
Got it, and agreed -- can you make it `localhost:8081` then?

Reply:
done